src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -647,33 +665,61 @@ public static function clean_cached_data() {
 		static::$i18n_schema              = null;
 	}
 
+	/**
+	 * Returns an array of all nested json files within a given directory.
```

Comment:
```suggestion
	 * Returns an array of all nested JSON files within a given directory.
```
Nitpick: Capitalize JSON here.